arithmetization/src/test/java/net/consensys/linea/zktracer/module/oob/OobCallTest.java

```diff
@@ -199,7 +200,8 @@ private void testCallSendValue(
         ToyExecutionEnvironmentV2.builder()
             .accounts(List.of(userAccount, contractCallerAccount, contractCalleeAccount))
             .transaction(tx)
-            .testValidator(x -> {})
+            .transactionProcessingResultValidator(
+                TransactionProcessingResultValidator.EMPTY_VALIDATOR)
```

Comment:
So, is this transaction supposed to fail --- or we just don't care?  If its supposed to fail then be good to make sure it does.

Reply:
I just preserve the same behaviour as before. Earlier it was `.testValidator(x -> {})` which is essentially same as empty validator. 

I am not sure what the test is intended for